data-collection/data-collection/Utilities/ImagePickerPermissions.swift

```diff
@@ -295,7 +295,14 @@ extension ImagePickerPermissions {
         func request(_ completion: @escaping (_ granted: Bool, _ shouldOpenSettings: Bool) -> Void) {
             PHPhotoLibrary.requestAuthorization { (status) in
                 DispatchQueue.main.async {
-                    completion(PHPhotoLibrary.authorizationStatus() == .authorized, false)
+                    var granted: Bool
```

Comment:
This value will be established once and as such we can specify this variable `let`.
```suggestion
                    let granted: Bool
```